server/src/logger/logger.go

```diff
@@ -26,7 +26,8 @@ func init() {
 // EnableDevelopmentLogger constructs a development logger and overwrites the default production logger
 // this is only for development
 func EnableDevelopmentLogger() {
-	loggerConfig := zap.NewDevelopmentConfig()
+	loggerConfig := zap.NewProductionConfig()
+	loggerConfig.EncoderConfig.StacktraceKey = "" // remove this line and add to init() for production
```

Comment:
Can you explain why this is needed here but not in production level? Since it's the only difference between the two

Reply:
If you mean the stacktrace, I wanted to disable the stacktrace to make the logs more readable for humans. Enabling the stacktrace leads to a very long and confusing log. But with the updated code, it's only disabled in production.